controls/cis_rhel10.yml

```diff
@@ -1138,7 +1142,11 @@ controls:
       levels:
           - l1_server
           - l1_workstation
-      status: pending
+      status: automated
+      rules:
+          - file_groupowner_cron_yearly
+          - file_owner_cron_yearly
```

Comment:
Please fix the CI issues:

```
Rule 'file_permissions_cron_yearly' must be in component 'cronie' because it's a member of 'cron_and_at' group.
Rule 'file_owner_cron_yearly' must be in component 'cronie' because it's a member of 'cron_and_at' group.
Rule 'file_groupowner_cron_yearly' must be in component 'cronie' because it's a member of 'cron_and_at' group.
```